src/Abstracts/MergeTag.php

```diff
@@ -212,6 +212,24 @@ public function set_resolver( $resolver ) {
 
 	}
 
+	/**
+	 * Sets resolver function
+	 *
+	 * @since 8.0.11
+	 *
+	 * @param array  $params merge tag configuration params.
+	 * @param string $property_name merge tag property name.
+	 * @param string $fallback merge tag fallback property name.
+	 */
+	public function set_property_name( array $params, string $property_name, string $fallback ) {
```

Comment:
Fix PHPStan errors

Reply:
In the end, I don't think this method should get all the merge tag constructor params, only the one which is needed.

Also, we should unify all the merge tags internal parameter name, not make it configurable